src/atomic.rs

```diff
@@ -496,7 +496,8 @@ impl<T> Atomic<T> {
     /// Bitwise "and" with the current tag.
     ///
     /// Performs a bitwise "and" operation on the current tag and the argument `val`, and sets the
-    /// new tag to the result. Returns the previous pointer.
+    /// new tag to the result. Returns the previous pointer.  Before the operation, `val` is
+    /// truncated to be fit in the bitmask returned by [`tag_bits()`](fn.tag_bits.html)).
```

Comment:
I see. In that case, my only complaints would be:

1. Parentheses don't match up: `(fn.tag_bits.html))` :) 
2. "to be fit in" -> "to fit into"

@jeehoonkang  You can merge the PR after that.